### ISSUE TITLE
Option to apply the obfuscate selection to all frames

### ISSUE THREAD

**Issue by cateyes99** [NickeManarin/ScreenToGif#1025]
For `Resize` & `Crop`, they apply to all frames. It'd be very convenient if there is an option to let us apply the obfuscate selection to all frames too😄

**Comment by cateyes99** [comment]
It might be more flexible to allow ranges of frames to be specified, like printing pages in **PDF**. For example,  _`1,2,11-50,90-`_ means applying the effect to the 1st, 2nd frames, and from the 11th to 50th frames, and all the rest from 90th frame.

**Comment by NickeManarin** [comment]
You can select multiple frames from the list. 
Like other features that depend on frame selection (caption, free text, drawings, shapes, etc), it will be applied to all selected frames.

**Comment by cateyes99** [comment]
Right, but let say if 15 frames per second, then 10 seconds would produce 150 frames. I'd be extremely tedious and time consuming to select all frames, or many of them just for a same action. If there is an option of specifying ranges it'd much more efficient and convenient.

**Comment by NickeManarin** [comment]
Yes, that would help too.
This option can be implemented for all features that depend on user selection.

**Comment by dlong500** [comment]
Any updates on this?  I love the software, but if you're making a video and need to obscure sensitive info it's basically useless if obfuscation needs to be done frame by frame.
